### DISCUSSION TITLE
Limit Issues submitting

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1339] [Ideas]
In issues, you state ''This issue tracker is not for support questions.'', but people can still ask support questions using the blank issue (not using any template). I don't know if it's possible, but u should prohibit opening blank issues so fewer people do it despite u prohibiting it.

**Comment by kagikn** [comment]
You can prevent contributors from opening a blank issue by [setting `blank_issues_enabled` to `false`](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) in your repositories, but I'm not planning to set this one, so contributors can easily report some focused issues such as ones for documentations or some security issues on our GitHub Actions. If too many people abuse the issue feature with blank issues, maybe I would but not right now.

**Reply by Michael21107** [reply]
gotcha, thank you
